web/chromecom.js

```diff
@@ -56,10 +55,12 @@ ChromeCom.request = function ChromeCom_request(action, data, callback) {
 /**
  * Resolves a PDF file path and attempts to detects length.
  *
- * @param {String} file Absolute URL of PDF file.
- * @param {Function} callback A callback with resolved URL and file length.
+ * @param {String} file - Absolute URL of PDF file.
+ * @param {OverlayManager} overlayManager - Manager for the viewer overlays.
+ * @param {Function} callback - A callback with resolved URL and file length.
  */
-ChromeCom.resolvePDFFile = function ChromeCom_resolvePDFFile(file, callback) {
+ChromeCom.resolvePDFFile =
+    function ChromeCom_resolvePDFFile(file, overlayManager, callback) {
```

Comment:
This fits on one line if you just use:
`ChromeCom.resolvePDFFile = function(file, overlayManager, callback) {`